doc/src/hugo/content/concepts/message.md

```diff
@@ -8,12 +8,12 @@ an implementation that is a message-driven system per the
 [Reactive Manifesto](https://reactivemanifesto.org). Messages in RIDDL are 
 a special case of an [aggregate type]({{< relref "type.md#aggregation" >}}) 
 and the _lingua franca_ of many RIDDL definitions. They define the API for:
-* [`adaptor`s]({{< relref "adaptor.md" >}})
```

Comment:
The s can be inside or outside the backticks but it should be consistent across all items; move it inside here or outside below on repositories

Reply:
It was consistently outside the backticks which is where it should be. we only want to use backticks around the actual RIDDL language keyword. Please revert.